src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const minDiscont = 3;
+  const baseDiscont = 7;
```

Comment:
Consider renaming `baseDiscont` to `baseDiscount` for better readability and to avoid confusion.